python/doc/examples/meta_modeling/general_purpose_metamodels/plot_expert_mixture.py

```diff
@@ -3,35 +3,33 @@
 ==================
 """
 # %%
-# In this example we are going to approximate a piece wise continuous function using an expert mixture of metamodels.
+# In this example we are going to approximate a piece-wise continuous function using an expert mixture of metamodels.
 #
-# The metamodels will be represented by the family of :math:`f_k \forall \in [1, N]`:
+# The metamodels will be represented by the family of :math:`f_k \forall k \in [\![1, n\!]`:
```

Comment:
The `stmaryrd` has commands for this purpose (see [ctan](https://ctan.org/pkg/stmaryrd?lang=en)):
```suggestion
# The metamodels will be represented by the family of :math:`f_k \forall k \in \llbracket 1, n \rrbracket`:
```
But please check that the `stmaryrd` package is available, otherwise this may fail. Perhaps a new macro may help, because there is no standard notation for integer intervals.